docs/source/nlp/nemo_megatron/batching.rst

```diff
@@ -5,17 +5,37 @@ Batching
 
 Batch size is one of the first parameters you should play with. For efficiency and convergence reasons we recommend you first try maximizing your batch size per GPU so that your GPU RAM usage is maximized.
```

Comment:
suggested revision

Batch size is one of the first parameters you should adjust. For efficiency and convergence, we recommend first maximizing your batch size per GPU to fully utilize your GPU RAM.